README.md

```diff
@@ -569,7 +569,39 @@ helper.setIndex('index_orderByPrice').search();
 #### Get the current index
 
 ```js
-var currentIndex = helper.state.index;
+var currentIndex = helper.getIndex();
+```
+
+### One time query
+
+Sometime it's convenient to reuse the current search parameters with small changes
+without changing the state stored in the helper. That's why there is a function
+called `searchOnce`. This method does not trigger `change` or `error` events.
+
+#### Using searchOnce with a callback
+
+```js
+var state = 
+helper.searchOnce(
+  helper.state.addFacetRefinement('facet', 'f2');,
```

Comment:
I think I would start with a simpler example instead of calling `addFacetRefinement`, something like:

```js
helper.searchOnce({hitsPerPage: 1}, function(error, content, state) { /*... */ });
```